mrp_mto_with_stock/models/mrp_production.py

```diff
@@ -80,19 +88,8 @@ def _adjust_procure_method(self):
                             move.product_uom_qty - qty_to_procure,
                         'unit_factor': move.unit_factor * (1 - ratio),
                     })
-                    move._action_confirm()
-                    move._action_assign()
-                elif qty_to_procure > 0.0:
-                    new_move = move
-                else:
-                    # If we don't need to procure, we reserve the qty
-                    # for this move so it won't be available for others,
-                    # which would generate planning issues.
-                    move._action_confirm()
-                    move._action_assign()
-            if new_move:
-                self.run_procurement(
-                    new_move, qty_to_procure, mto_with_no_move_dest_id)
```

Comment:
why did you remove all this logic?

Reply:
In the current version, this logic create a duplication of the procurement